### ISSUE TITLE
Docs: README for eslint-config-next just to link to ESLint rules

### ISSUE THREAD

**Issue by sm-kc** [vercel/next.js#50843]
### What is the improvement or update you wish to see?

This is a very minor thing. As a brand new user of Next.js, I saw that it comes with an ESLint configuration, so from habit I went to [its package page](https://www.npmjs.com/package/eslint-config-next). However `eslint-config-next` doesn't have a README so that was a dead end. It would have saved me having to go search for the docs page (linked below) if there had been a README with a link to it.

### Is there any context that might help us understand?

Just smoothing the path for new users of Next.js who don't know their way around the Next.js docs.

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/pages/building-your-application/configuring/eslint

<sub>[DX-1696](https://linear.app/vercel/issue/DX-1696/docs-readme-for-eslint-config-next-just-to-link-to-eslint-rules)</sub>

**Comment by JesseKoldewijn** [comment]
I agree that there should be a README but to be fair, the docs does include all the details about the eslint-config-next package.

**Comment by manovotny** [comment]
Good suggestion @sm-kc. I created a PR to add a README (https://github.com/vercel/next.js/pull/51059). However, it's sister package, [eslint-plugin-next](https://www.npmjs.com/package/eslint-plugin-next) already [has a README](https://github.com/vercel/next.js/tree/canary/packages/eslint-plugin-next) and it doesn't show up on NPM. 

@ijjk @timneutkens @styfle is there any way we can get these READMEs published with the package so NPM can link to the documentation?

**Comment by styfle** [comment]
> However, it's sister package, [eslint-plugin-next](https://www.npmjs.com/package/eslint-plugin-next) already [has a README](https://github.com/vercel/next.js/tree/canary/packages/eslint-plugin-next?rgh-link-date=2023-06-09T18%3A32%3A41Z) and it doesn't show up on NPM.

That's because its no longer maintained. It hasn't been published in 3 years as seen on the [Versions](https://www.npmjs.com/package/eslint-plugin-next?activeTab=versions) tab.

<img width="778" alt="image" src="https://github.com/vercel/next.js/assets/229881/78cc9e80-7092-42e9-9927-d8485389f826">

Looks like it was renamed to [`@next/eslint-plugin-next`](https://github.com/vercel/next.js/blob/2bc10616096500ff7664d25d16695f640beb8538/packages/eslint-plugin-next/package.json#LL2C12-L2C36) in package.json

**Comment by manovotny** [comment]
> That's because its no longer maintained. It hasn't been published in 3 years as seen on the [Versions](https://www.npmjs.com/package/eslint-plugin-next?activeTab=versions) tab.

@styfle should we run `npm deprecate <package-spec> <message>` on `eslint-plugin-next` then and have the message point to `@next/eslint-plugin-next`? That would automatically put at banner on top of NPM's page.

**Comment by styfle** [comment]
@manovotny Its not owned by Next.js

You'll have to talk to @Timer 

<img width="163" alt="image" src="https://github.com/vercel/next.js/assets/229881/9a350af1-6ae0-417d-8be5-46082cb9f4ba">

**Comment by scottdotjs** [comment]
Commenting to take ownership, @sm-kc was my account at a former employer.

**Comment by samcx** [comment]
Closing this issue since a `homepage` was added, which I believe is adequate enough instead of a dedicated README.md → https://github.com/vercel/next.js/pull/51059#pullrequestreview-1473042530.
